net/rsync/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=rsync
-PKG_VERSION:=3.2.4
+PKG_VERSION:=3.2.5pre2
 PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://download.samba.org/pub/rsync/src
-PKG_HASH:=6f761838d08052b0b6579cf7f6737d93e47f01f4da04c5d24d3447b7f2a5fad1
+PKG_SOURCE_URL:=https://codeload.github.com/WayneD/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
```

Comment:
I don't like this change, why we can not use https://download.samba.org/pub/rsync/src-previews/rsync-3.2.5pre2.tar.gz ?

Reply:
Initially, I did not see the pre-release on samba.org but I modified per your suggestion.